.github/workflows/assign-dependabot-reviewer.yml

```diff
@@ -0,0 +1,43 @@
+name: Assign Reviewer for Dependabot
+
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+    branches:
+      - main
+  issue_comment:
+    types: [ created ]
```

Comment:
what's this for?

Reply:
This allows the workflow to be restarted (in case there's a failure or some other reason) instead of waiting for dependabot to synchronize the PR. The comment would need to contain `/dependabot-reassign`